src/datetime.rs

```diff
@@ -466,13 +398,13 @@ impl DateTime {
     /// let dt_utc_plus2 = dt_z.in_timezone(7200).unwrap();
     /// assert_eq!(dt_utc_plus2.to_string(), "2000-01-01T17:00:00+02:00");
     /// ```
-    pub fn in_timezone(&self, offset: i32) -> Result<Self, ParseError> {
-        if offset.abs() >= 24 * 3600 {
+    pub fn in_timezone(&self, tz_offset: i32) -> Result<Self, ParseError> {
+        if tz_offset.abs() >= 24 * 3600 {
```

Comment:
same.

Reply:
I might be wrong but I think in the above case it was possible, since it called `Time` functions.
But in this one it just constructs the DateTime from total seconds so the validation should stay.

Reply:
ok.